VideoWeb/VideoWeb.EventHub/Handlers/RecordingConnectionEventHandler.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using EventType = VideoWeb.EventHub.Enums.EventType;
+
+namespace VideoWeb.EventHub.Handlers
+{
+    public class RecordingConnectionEventHandler(
+        IHubContext<Hub.EventHub, IEventHubClient> hubContext,
+        IConferenceService conferenceService,
+        ILogger<EventHandlerBase> logger)
+        : EventHandlerBase(hubContext, conferenceService, logger)
+    {
+
+        public override EventType EventType => EventType.RecordingConnectionFailed;
+
+        protected override async Task PublishStatusAsync(CallbackEvent callbackEvent)
+        {
+            var conferenceId = SourceConference.Id;
+
+            var participantId = callbackEvent.ParticipantId;
+
+            var message = $"Recording Connection Failed: Conference Id: {conferenceId} - Participant id: {participantId}";
+            Logger.LogTrace(
```

Comment:
```suggestion
            Logger.LogTrace("Recording Connection Failed: Conference Id: {ConferenceId} - Participant id: {ParticipantId}", conferenceId, participantId);
```

Structured logging, as the suggested implementation shown, allows log tools to better parse and analyse log data

Reply:
this is sonar suggestion

Reply:
no, this is my suggestion. With all the recent work we need to make sure we use structured logging

Reply:
even if it is a sonar suggestion, it's a good one to implement

Reply:
Sorry I was not clear. The change I did is a Sonar suggestion. Your suggested change is flagged from Sonar as a Maintainability issue.

Reply:
However I have re-changed the log

Reply:
Cool I can see the change and Sonar is happy with it :)